specification/paths/Collections.json

```diff
@@ -0,0 +1,197 @@
+{
+  "get": {
+    "tags": [
+      "Collections"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "shipments.manage",
+          "shipments.view"
+        ]
+      }
+    ],
+    "summary": "Get all available collections.",
+    "description": "This endpoint retrieves collections available to your API client.",
+    "parameters": [
+      {
+        "name": "include",
+        "in": "query",
+        "description": "Comma separated string of the relationship names you want to include the data of.<br>The relationships that can be included are:<ul><li>`contract`</li><li>`contract.carrier`</li><li>`manifest`</li><li>`shop`</li><li>`status`</li></ul>",
+        "schema": {
+          "type": "string"
+        }
+      }
+    ],
+    "responses": {
+      "200": {
+        "description": "Retrieved the collections.",
+        "content": {
+          "application/vnd.api+json": {
+            "schema": {
+              "type": "object",
+              "required": [
+                "data",
+                "meta",
+                "links"
+              ],
+              "properties": {
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "allOf": [
+                      {
+                        "$ref": "#/components/schemas/CollectionResponse"
+                      },
+                      {
+                        "properties": {
+                          "attributes": {
+                            "properties": {
+                              "tracking_code": {
+                                "type": "string",
+                                "example": "TRC123456"
+                              }
+                            }
+                          }
+                        }
+                      }
```

Comment:
I would prefer this attribute to be part of the `Collection` schema (read-only) and indicate the presence of the value in the description. Because now we have these 12 lines duplicated in 3 different places + if I would search for a complete list of Collection resource properties, I would check the collection.json schema and not expect additional properties in response paths.

I see why you want to do it this way though, but our specification is structured in another way. There are many attributes and especially relationships which are not present or even able to have a value in a POST response. They are just possible properties of the particular **resource** being created, so all endpoints simply describe that resource and the properties that definitely have a value.

Let's check with the team tomorrow. If we want to be this explicit, we should modify other schemas as well.